block-non-tracking-third-party-cookies/config_reference.json

```diff
@@ -0,0 +1,55 @@
+{
+    "readme": "This configuration has only the nonTracking3pCookies feature",
+    "features": {
+        "nonTracking3pCookies": {
+            "settings": {
+                "excludedCookieDomains": [
+                    {
+                        "domain": "excluded-cookie-domain.com",
```

Comment:
I think we should use different domain than in `cookie` to make sure that for whatever reason `cookie` exception is not used when we expect `nonTracking3pCookies` exception.